src/components/Speciality/Talks.js

```diff
@@ -14,7 +14,7 @@ const TalksSection = ({ talks: allTalks }) => {
   const featured = allTalks.find(({ featured }) => featured)
   const cta = allTalks.find(({ cta }) => cta)
 
-  return talks && talks.length ? (
+  return (
```

Comment:
Why did you remove `talks && talks.length`? Could have been used as `talks && talks.length && <BlueBackground>` with `talks = []` in props

Reply:
Technically due to the updates in the `speciality-component` we don't need these but you're right they should still be localised here if we're not setting any defaults as well